src/schema/v2/artist/helpers.ts

```diff
@@ -237,6 +237,80 @@ export const getAuctionRecord = async (artist, auctionLotsLoader) => {
   }
 }
 
+export const enrichWithArtistCareerHighlights = async (
+  kind,
+  artist,
+  artistCareerHighlightsLoader
+) => {
+  if (!artistCareerHighlightsLoader) return
+
+  const validTypes = ["SOLO_SHOW", "GROUP_SHOW", "COLLECTED"]
+  const invalidTypes = kind.filter((type) => !validTypes.includes(type))
+
+  if (invalidTypes.length) {
+    console.warn(
+      `[schema/v2/artist/insights] invalid artist insight types: ${invalidTypes}`
+    )
+
+    return
+  }
+
+  if (kind.includes("SOLO_SHOW")) {
+    const soloShows = await getArtistCareerHighlights(
+      artistCareerHighlightsLoader,
+      artist._id,
+      "solo"
+    )
+
+    /* eslint-disable-next-line require-atomic-updates */
+    artist.soloShowInstitutions = soloShows
```

Comment:
maybe just:
```jsx
artist.soloShowInstitutions = await getArtistCareerHighlights(
      artistCareerHighlightsLoader,
      artist._id,
      "solo"
    )
```
declaring the const is maybe redundant, right?

Reply:
yes!